src/sections/Learn-Layer5/Course-Overview/index.js

```diff
@@ -81,6 +91,13 @@ const CourseOverview = ({ course, chapters, serviceMeshesList }) => {
             title="Get Started"
             url={`istio/${course.frontmatter.toc[0]}`}
           />
```

Comment:
Another good thing will be, to update the title of this button to `Start again` or something similar (@leecalcote might help with this) if there is already a bookmark (a check similar to the `resume` button would do`).

Reply:
Keep this one as `Resume` only.
Make the above button `Start Again` when there is a bookmark.
So, there will be 2 cases:
1. When  a new user comes he/she should see only `Get Started` button
2. When a returning user comes, he/she should see `Start again` and `Resume` button.

Reply:
Adjust the code to accommodate both of the above cases.

Reply:
Oh sorry. Got it. I will make the necessary changes.